mitodl/odl-video-service
# More JS upgrades

#### What are the relevant tickets?
N/A

#### What's this PR do?
- Updates more JS dependencies, including major ones like node, webpack and yarn
- Makes some changes to webpack configuration files so they still work with the latest version
- Fixes tests that broke after the upgrades
- Adjusts file formatting that was no longer valid after the upgrades

#### How should this be manually tested?
- Rebuild all instances.  Delete `/static/bundles` and `staticfiles/bundles`
- Tests should pass, functionality should still work (video upload, playback, etc).
- Run `docker-compose run watch node node_modules/webpack/bin/webpack.js --config webpack.config.prod.js --bail`, after it's done there should be a bunch of files under `static/bundles`


There's likely more that could still be updated, but I'm inclined to leave that for a separate PR
